src/public/app/widgets/note_type.js

```diff
@@ -35,7 +35,7 @@ const TPL = `
         overflow-x: hidden;
     }
     </style>
-    <button type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="btn btn-sm dropdown-toggle note-type-button">
+    <button type="button" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="btn btn-sm dropdown-toggle note-type-button">
```

Comment:
Note type separator not visible:

<img width="189" alt="image" src="https://github.com/user-attachments/assets/6db7e151-8c93-497e-9243-25b2298ec8b1">

Compare with:

<img width="189" alt="image" src="https://github.com/user-attachments/assets/52f0b7ff-7674-42eb-9862-f94796993119">